packages/block-library/src/navigation-menu/edit.js

```diff
@@ -51,11 +51,18 @@ function NavigationMenu( {
 			if ( ! pages ) {
 				return null;
 			}
-			return pages.map( ( page ) => {
-				return [ 'core/navigation-menu-item',
-					{ label: page.title.rendered, url: page.permalink_template },
-				];
-			} );
+
+			return pages.map( ( { title, permalink_template: destination, type, link, id } ) => (
+				[ 'core/navigation-menu-item', {
+					label: title.rendered,
+					destination,
```

Comment:
This is the default behavior of the menu: showing initially the top level pages.

Reply:
Oh, I should have been more specific. Do we need the `destination` parameter still? I can't find a reference to it anywhere

Reply:
Just FYI, `destination` comes from the `page.permalink_template`, while on the other hand `url` comes from `page.link` 👏 :devs:

Reply:
My understanding is that this should map to the navigation-menu-item block's attributes. There doesn't seem to be a `destination` attribute.

Reply:
> My understanding is that this should map to the navigation-menu-item block's attributes. There doesn't seem to be a destination attribute.

@talldan You are correct. There isn't a `destination` attribute. However, there _is_ a `permalink_template` attribute. You will see `destination` is used as an alias here to avoid eslint camel case rules firing off.

Reply:
I also can't see `destination` being used in [the Nav Menu Item block implementation](https://github.com/WordPress/gutenberg/pull/18062/files#diff-5f9b4b45b9379c94d90e6e418727bfa0R12). Is this a defunct attribute @retrofox?